### PR TITLE
Papa 8a and Irminger 8 updates

### PR DESCRIPTION
**Pull request by cdobs**
GL276 updates can be reviewed in Papa 8a QLR (3203-00805) and FCP (3304-00160-00056)
GL469 updates can be reviewed from FMC Annotation (attached here)
image (4)

### CODE REVIEW

File: deployment/GI05MOAS-GL469_Deploy.csv [hunk 1]
```diff
@@ -7,7 +7,7 @@ AR46,,AR46,,GI05MOAS-GL469-00-ENG000000,3,1,2020-08-15T17:21:00,2020-08-16T00:11
...
-AR60-01,,,,GI05MOAS-GL469-02-DOSTAM000,4,1,2021-08-11T04:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00168,59.85902,-39.12405,,1000,1000,top mount,,
-AR60-01,,,,GI05MOAS-GL469-04-CTDGVM000,4,1,2021-08-11T04:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,59.85902,-39.12405,,1000,1000,,,
\ No newline at end of file
+AR60-01,,UNRECOVERED,,GI05MOAS-GL469-00-ENG000000,4,1,2021-08-11T04:04:00,2021-10-19T04:14:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGCON-GLDROO-00469,59.85902,-39.12405,,1000,1000,glider lost at sea,,
+AR60-01,,UNRECOVERED,,GI05MOAS-GL469-01-FLORDM000,4,1,2021-08-11T04:04:00,2021-10-19T04:14:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-FLORDM-03528,59.85902,-39.12405,,1000,1000,glider lost at sea,,
+AR60-01,,UNRECOVERED,,GI05MOAS-GL469-02-DOSTAM000,4,1,2021-08-11T04:04:00,2021-10-19T04:14:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00168,59.85902,-39.12405,,1000,1000,top mount; glider lost at sea,,
+AR60-01,,UNRECOVERED,,GI05MOAS-GL469-04-CTDGVM000,4,1,2021-08-11T04:04:00,2021-10-19T04:14:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,59.85902,-39.12405,,1000,1000,glider lost at sea,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL469_Deploy.csv:13]: looks good

File: deployment/GP05MOAS-GL276_Deploy.csv [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+SR1811,,TY-1902,,GP05MOAS-GL276-04-CTDGVM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-00104,50.1295,-144.5424,,1000,1000,,,
+SU20211112,,SU20211112,,GP05MOAS-GL276-00-ENG000000,3,1,2021-11-13T09:25:00,2021-11-13T13:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGCON-GLDROO-00276,44.78732,-125.31298,,1000,1000,short deployment due to steering issues,,
+SU20211112,,SU20211112,,GP05MOAS-GL276-01-FLORDM000,3,1,2021-11-13T09:25:00,2021-11-13T13:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORDM-03583,44.78732,-125.31298,,1000,1000,short deployment due to steering issues,,
+SU20211112,,SU20211112,,GP05MOAS-GL276-02-DOSTAM000,3,1,2021-11-13T09:25:00,2021-11-13T13:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00398,44.78732,-125.31298,,1000,1000,top mount; short deployment due to steering issues,,
+SU20211112,,SU20211112,,GP05MOAS-GL276-04-CTDGVM000,3,1,2021-11-13T09:25:00,2021-11-13T13:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-09238,44.78732,-125.31298,,1000,1000,short deployment due to steering issues,,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-GL276_Deploy.csv:13]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/GI05MOAS-GL469_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/GI05MOAS-GL469_Deploy.csv:13]: Looks good.   So sad.

File: deployment/GP05MOAS-GL276_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/GP05MOAS-GL276_Deploy.csv:13]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**